plugin/rewrite/edns0.go

```diff
@@ -274,14 +275,22 @@ func (rule *edns0VariableRule) ruleData(w dns.ResponseWriter, r *dns.Msg) ([]byt
 		return rule.portToWire(port)
 	}
 
+	// Metadata variable
+	ctxVarName := strings.Trim(rule.variable, "{}")
+	ctxVal := ctx.Value(ctxVarName)
+	if b, ok := ctxVal.([]byte); ok {
+		// Metadata value is written as ascii string
```

Comment:
instead of writing ASCII,  can we write raw (i.e.  []byte)  on the data in the context. 
Leave the format of the data ( asccii or byte ...)  to the plugin that save the data into the metadata.

Reply:
It is initially []byte. It is just an not accurate comment.